packages/csv-parser-discount-code/src/main.js

```diff
@@ -79,16 +85,25 @@ export default class CsvParserDiscountCode {
   // Convert the cartDiscounts field to an array of references to commercetools
   // cartDiscounts
   _cartDiscountsToArray(item: Object) {
-    if (item.cartDiscounts) {
-      const cartDiscounts = item.cartDiscounts
-        .split(this.multiValueDelimiter)
-        .map(cartDiscount => ({
-          typeId: 'cart-discount',
-          id: cartDiscount,
-        }))
-      return Object.assign(item, { cartDiscounts })
-    }
-    return item
+    const { cartDiscounts, ...rest } = item
+
+    return cartDiscounts
+      ? Object.assign(rest, {
+          cartDiscounts: cartDiscounts
+            .split(this.multiValueDelimiter)
+            .map(cartDiscount => ({
```

Comment:
cartDiscount -> cartDiscountId

Reply:
this wouldn't work as I'm returning the object

Reply:
the point is that you should rename this.
you do assign `cartDiscount` as id in the code below.
when I read `cartDiscount`, I think of the entity, not the id